numba/tests/support.py

```diff
@@ -878,3 +880,38 @@ def check_warnings(self, messages, category=RuntimeWarning):
                     self.assertEqual(w.category, category)
                     found += 1
         self.assertEqual(found, len(messages))
+
+
+def _format_jit_options(**jit_options):
+    if not jit_options:
+        return ''
+    out = []
+    for key, value in jit_options.items():
+        if isinstance(value, str):
+            value = '"{}"'.format(value)
+        out.append('{}={}'.format(key, value))
+    return ', '.join(out)
+
+
+@contextlib.contextmanager
+def create_temp_module(source_lines, **jit_options):
```

Comment:
Suggest adding a doc string now this is a formal utility.

Reply:
@stuartarchibald 
done
thanks

Reply:
thanks